### ISSUE TITLE
Infinite loop during install > 1.8.4.

### ISSUE THREAD

**Issue by mkrstat** [vendure-ecommerce/vendure#1910]
**Describe the bug**
Potential infinite loop during install with >1.8.4.

**To Reproduce**
Install via `npx @vendure/create my-app --log-level verbose` 

**Expected behavior**
Unusually long installing times > 45min, seemingly looping at `extract:@vendure/admin-ui-plugin: sill extract @vendure/admin-ui-plugin@latest extracted to /` (semi-regularly shown with the verbose log-level and starting to loop after about 4min of installing)

**Environment (please complete the following information):**
Vendure 1.8.5. on Ubuntu 20.04 with Node v14.21.1 and npm 6.13.6; MySQL database.

**Additional context**
Maybe just me being impatient? Either way, using `npx @vendure/create@1.8.4 my-app --log-level verbose` worked after about 10min of installing.

**Comment by michaelbromley** [comment]
Hi,
45 minutes is certainly longer than usual. Even 10 minutes is quite long. Any potential issue that arises during the package installation phase might be related to temporary network connectivity issues, or other environmental factors (load on your computer, disk space etc).

**Comment by mkrstat** [comment]
I've been using a basic digitalocean droplet for what it's worth - have been trying to install Vendure for the past few hours actually, haha.

It seems rather particular that it spends more than 30 minutes solely on what seems to be extracting the admin-ui-plugin? Specifically because there is no freezing.

**Comment by michaelbromley** [comment]
Are you running `npx @vendure/create my-app` directly on your DO droplet?

If so, and considering you are on the basic droplet size, I would not be surprised if you are running into resource limits (CPU or RAM). You can check this by looking at resource usage charts for that droplet in your DO dashboard.

**Comment by mkrstat** [comment]
What do you mean with "directly"? - I just access the server through PuTTY. CPU usage during installation is around 85-100%, little writing (~ 0,1-2 MB/s), intense reading (200-400 MB/s) on the disks.

**Comment by mkrstat** [comment]
What do you mean with "directly"? - I just access the server through PuTTY. CPU usage during installation is around 85-100%, little writing (~ 0,1-2 MB/s), intense reading (200-400 MB/s) on the disks.

(Sorry, didn't mean to close with the comment, misclicked.)

**Comment by michaelbromley** [comment]
Yeah you can add a comment to the other issue. There is a work-around linked there too.
